### ISSUE TITLE
Upgrade script sometimes failed

### ISSUE THREAD

**Issue by alexyangjie** [v2ray/v2ray-core#2373]
1) What version of V2Ray are you using (If you deploy different version on server and client, please explicitly point out)?
v4.22.1

2) What's your scenario of using V2Ray? E.g., Watching YouTube videos in Chrome via Socks/VMess proxy.
General day-to-day use

3) What did you see? (Please describe in detail, such as timeout, fake TLS certificate etc)
The upgrade script failed sometimes. I have pinpoint the issue to this command to obtain the new version. (https://github.com/v2ray/v2ray-core/blob/2227557b86b233c10e5e92f59f45ba76fcd0ac2c/release/install-release.sh#L258)
```
curl -s "https://api.github.com/repos/v2ray/v2ray-core/releases/latest" --connect-timeout 10| grep 'tag_name' | cut -d\" -f4
```
Sometimes it prints out the correct version, i.e. `v4.23.1`. But sometimes it prints out this `https://api.github.com/repos/v2ray/v2ray-core/releases/24822244`, which leads to the download failure. 

4) What's your expectation?
It always gets the correct latest version. 

5) Please attach your configuration file (**Mask IP addresses before submit this issue**).
N/A

6) Please attach error logs, especially the bottom lines if the file is large. Error log file is usually at `/var/log/v2ray/error.log` on Linux.
N/A

7) Please attach access log. Access log is usually at '/var/log/v2ray/access.log' on Linux.
N/A

8) Other configurations (such as Nginx) and logs.
N/A

9) If V2Ray doesn't run, please attach output from `--test`.
N/A

10) If V2Ray service doesn't run, please attach journal log.
N/A

**Comment by kslr** [comment]
fix #2374

**Comment by alexyangjie** [comment]
That works like magic! Thank you for the quick fix.
